_posts/2022-12-05-Introducting-Identity.markdown

```diff
@@ -0,0 +1,35 @@
+# Introducing Identity for OpenSearch
+
+OpenSearch handles critical data; how that data is permissioned and how a cluster is operated is a Big Deal™.  Consider the question "Who can delete all the indices in the cluster?", in the core of OpenSearch, the answer is whoever can make a request to the cluster.  
+
+Within the OpenSearch Project, there is a plugin Security that allows finer levels of access control providing a more satisfying answer.  Security uses the plugin architecture of OpenSearch that forces concepts around identity to be optional in core, plugins, and extensions components.
+
+Let’s permeate identity in every aspect of OpenSearch – enabling security by default - everywhere.
+
+## What are OpenSearch’s identities?
+In the literature, an identity is an Authenticated [1] Subject [2].  In the systems of OpenSearch, an identity embodies sources of interaction.
+- Users make REST API calls in interactive sessions
+- Node-to-node communication in a cluster
+- Background jobs interactions
+- Plugins in-process interactions
+- Extensions actions extra-process interactions
+
+![image](https://user-images.githubusercontent.com/2754967/201424268-5687f162-4857-424b-96b8-ca8aff5f649a.png)
+
+## Why have identity in all OpenSearch systems?
+By knowing who is performing actions we can check if that user is Authorized [3].  Plugins and extensions can only do actions that have been permitted to their identity.  The existing security plugin does not have the ability to control what other plugins can do.

```

Comment:
I am wondering if we should keep consistent with subject vs user as we often have debate on identity PRs around the topic too. 

To make the distinction clear do you think it makes sense to have a sentence of 2 somewhere in this post along the lines of: 

```User and Subject are often used interchangeably. For this blog post I will use the term Subject to represent a generalized user. For example, when a user queries OpenSearch, their subject will be associated with a list of permissions and evaluated before performing the query to determine if the action is authorized. In the model we are proposing, extensions will also be associated with a Subject. Administrators will be able to assign permissions to extensions to isolate how an extension can interact with OpenSearch.```

Reply:
This is a much better rewrite, unfortunately I pulled all writing about Subjects from the post, but we will get back to this in future examples